pokemon.js

```diff
@@ -0,0 +1,75 @@
+let section =document.getElementById("container");
+let whencl=document.getElementById("whencl");
+//for searchbar
+const searchInput = document.querySelector('.input');
+let arr=[];
+searchInput.addEventListener('input',(input)=>{
+    let value=input.target.value;
+    const filteredEle=arr.filter(ele => {
+        let isVisible=ele.name.includes(value);
+       return isVisible;
```

Comment:
isVisible variable is redundent, we can just return ele.name.includes(value); (inline variable)